components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -69,16 +80,30 @@ pub struct TimeSymbolsV1<'data> {
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[yoke(prove_covariance_manually)]
 pub struct Eras<'data> {
+    /// Symbol data for era names.
```

Comment:
Nit: Specify what the keys and values in the map are. I think they are era codes to era display names. You can verify by looking at the JSON data files.

Reply:
Done.